### PR TITLE
LPS-149128 Add integration test for boost longer contents element

### PR DESCRIPTION
**Pull request by gustavolimav**
https://issues.liferay.com/browse/LPS-149128

### CODE REVIEW

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:stable

**Comment by gustavolimav** [comment]
ci:test:sf
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:stable

**Comment by gustavolimav** [comment]
ci:test:relevat

**Comment by gustavolimav** [comment]
ci:test:relevant

**Comment by gustavolimav** [comment]
ci:test:search

**Comment by gustavolimav** [comment]
ci:test:stable

File: modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java [hunk 1]
```diff
@@ -219,6 +219,34 @@ public void testBoostFreshness() throws Exception {
...
+					"factor", 1.5
+				).put(
+					"modifier", "ln"
+				).build()
+			},
+			new String[] {"Boost Longer Contents"});
+
+		_keywords = "alpha";
+
+		_assertSearch("[beta alpha, alpha alpha, charlie alpha]");
```
Comment (joshuacords) [inline modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java:381]: This seems to indicate that we boosted shorter contents to go first. @gustavolimav, could you explain?

**Comment by joshuacords** [comment]
@gustavolimav, could you link all the tickets this PR addresses for easier review?

File: modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java (hunk 1 above)
Comment (gustavolimav) [reply modules/dxp/apps/search-experiences/search-experiences-test/src/testIntegration/java/com/liferay/search/experiences/internal/blueprint/test/SXPBlueprintSearchResultTest.java:381]: @joshuacords 

`_setUpJournalArticles(
			new String[] {"alpha alpha", "", ""},
			new String[] {"beta alpha", "charlie alpha", "alpha alpha"});`

When I create the journal articles the only article with content is "beta alpha", so when we use the boost longer contents element, "beta alpha" is supposed to be the highest score, and if we don't use the element the article with the highest score is "alpha alpha".

Probably it's better to add another article with content to prove that it's really working.

**Pull request closed without merging**